front/src/styles/scss/applications/operationalStudies/_boardWrapper.scss

```
@@ -29,11 +32,44 @@
 
     .board-header-button {
       color: var(--grey30);
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 24px;
+      height: 16px;
+
+      &:focus {
+        outline: none;
+      }
+
+      &.menu-is-opened {
+        color: var(--grey50);
+        outline: none;
+        box-shadow:
+          0 1px 2px 0 rgba(0, 0, 0, 0.16),
+          0 2px 2px -1px rgba(255, 171, 88, 0.27);
+        border-radius: 4px;
+        background-color: rgba(255, 255, 255, 1);
```

nit : we try to use design system variables whenever possible
```suggestion
        background-color: var(--white100);
```

---

Done!